tests/nested/manual/fde-on-classic/task.yaml

```diff
@@ -0,0 +1,85 @@
+summary: Check that a FDE based classic image can be booted
+
+details: |
+  This test creates a classic image that looks like what the installer
+  would create and we boot into it.
+
+systems: [ubuntu-22.04-64]
+
+environment:
+  NESTED_ENABLE_SECURE_BOOT: false
+  NESTED_USE_CLOUD_INIT: true
+  CACHE_D: ./cache
+
+prepare: |
+  # build gadget snap with gadget.yaml matching disk structure of the image
+  VERSION="$(tests.nested show version)"
+  snap download --basename=pc --channel="$VERSION/edge" pc
+  unsquashfs -d pc-gadget pc.snap
+  cp gadget.yaml pc-gadget/meta/
+  snap pack --filename=pc_x1.snap pc-gadget
+
+  # create an image that looks like a classic image
+  # Note that "mk-image" is left as a script so that it can also be
+  # run outside of spread easily for quick interactive testing
+  ./mk-image.sh ./boot.img "$CACHE_D" "./mnt" ./pc_x1.snap
+  # replaces snap-bootstrap in initramfs and snapd in rootfs
+  # TODO chroot install of deb? /home/gopath/src/github.com/snapcore/snapd_*_amd64.deb
```

Comment:
(nitpick) I think this comment no longer applies, inside `replace-image-files.sh` there is a call `sudo chroot "$MNT"/data apt install -y --no-install-recommends ./snapd.deb` so this is happening - unless I misunderstand something of course.